tests/js/comments-closure-typecast/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -372,9 +478,11 @@ foo = /** @type {!Foo} */ (/** @type {!Baz} */ (baz).bar);
 
 const BarImpl = /** @type {BarConstructor} */ (
   /** @type {unknown} */
-  (function Bar() {
-    throw new Error("Internal error: Illegal constructor");
-  })
+  (
+    function Bar() {
+      throw new Error("Internal error: Illegal constructor");
+    }
+  )
```

Comment:
minor but also looks unintended?

Reply:
Something to improve in the future. The idea was to format type assertions similarly to function calls while preserving at the same the usual behavior of comments in Prettier. If a comment is on its own line, Prettier doesn't change that. But now I see that the behavior of the parens could depend on this condition (whether the comment is own-line) too.